nemo/collections/nlp/data/machine_translation/machine_translation_dataset.py

```diff
@@ -0,0 +1,260 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Pytorch Dataset for training Neural Machine Translation."""
+
+from collections import OrderedDict
+from dataclasses import dataclass
+
+import numpy as np
+from omegaconf.omegaconf import MISSING
+
+from nemo.collections.nlp.data.data_utils.data_preprocessing import dataset_to_ids
+from nemo.core import Dataset
+
+__all__ = ['TranslationDataset']
+
+
+@dataclass
+class TranslationDataConfig:
+    src_file_name: str = None  # MISSING
+    tgt_file_name: str = None  # MISSING
```

Comment:
Why `None` and not `MISSING`?

Reply:
Yup this should be missing unless code can handle none case